documentation/assemblies/tracing/assembly-distributed-tracing.adoc

```diff
@@ -21,15 +26,14 @@ Support for tracing is built in to the following components:
 
 You enable and configure tracing for these components using template configuration properties in their custom resources.
 
-To enable tracing in Kafka producers, consumers, and Kafka Streams API applications, you _instrument_ application code using the link:https://github.com/opentracing-contrib/java-kafka-client/blob/master/README.md[OpenTracing Apache Kafka Client Instrumentation^] library (included with Strimzi). When instrumented, clients generate trace data; for example, when producing messages or writing offsets to the log.
+To enable tracing in Kafka producers, consumers, and Kafka Streams API applications, you _instrument_ application code.
+When instrumented, clients generate trace data; for example, when producing messages or writing offsets to the log.
```

Comment:
"writing offsets to the log" ?? What does it mean? Trace data are available when producing and consuming messages but not for example when committing offsets (if this is what you meant).